docs/docs/modeling.md

```diff
@@ -110,6 +110,30 @@ classDiagram
 
 ## Task - タスク
 
+### Task
+
+- A specific work or activity that tenants are responsible for performing.
+
+### taskId
+
+- TaskId is the UUID
+
+- Constraints:
```

Comment:
You might want to remove this section as there is no specific constraint for `taskId`.